cf-custom-resources/lib/backlog-per-task-calculator.js

```diff
@@ -0,0 +1,123 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+"use strict";
+
+const aws = require("aws-sdk");
+
+// AWS Clients that are overriden in tests.
+let ecs, sqs;
+
```

Comment:
We don't even need a report function?

Reply:
Yeah! Because this Lambda won't be invoked by a Custom Resource, it's instead going to be invoked every minute by a CloudWatch event so we don't need it.

Reply:
gotcha!